bundles/org.openhab.binding.groheondus/src/main/java/org/openhab/binding/groheondus/internal/handler/GroheOndusAccountHandler.java

```diff
@@ -120,46 +121,47 @@ public void dispose() {
         if (refreshTokenFuture != null) {
             refreshTokenFuture.cancel(true);
         }
+        super.dispose();
     }
 
-    @Override
-    public void initialize() {
+    private boolean login() {
         GroheOndusAccountConfiguration config = getConfigAs(GroheOndusAccountConfiguration.class);
-
         if (this.accountServlet == null) {
             this.accountServlet = new AccountServlet(httpService, this.getThing().getUID().getId(), this);
         }
 
         if ((config.username == null || config.password == null) && !this.hasRefreshToken()) {
             updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_PENDING,
                     "Need username/password or refreshToken");
```

Comment:
Consider moving text to src/main/resources/OH-INF/i18n/groheondus.properties to allow for localization.